services/webnn/ort/graph_builder_ort.cc

```diff
@@ -676,19 +669,13 @@ GraphBuilderOrt::AddInstanceNormalizationOperation(
       case OperandDataType::kFloat16: {
         std::vector<uint16_t> scale_data_fp16 =
             ConvertFloat32ToFloat16(scale_data);
-        scale_name = CreateInitializerAsRawData(
-            constant_dims,
-            base::span(reinterpret_cast<const uint8_t*>(scale_data_fp16.data()),
-                       sizeof(uint16_t) * scale_data_fp16.size()),
-            input_data_type);
+        scale_name = CreateInitializer<uint16_t>(constant_dims, scale_data_fp16,
+                                                 input_data_type);
         break;
       }
       case OperandDataType::kFloat32: {
-        scale_name = CreateInitializerAsRawData(
-            constant_dims,
-            base::span(reinterpret_cast<const uint8_t*>(scale_data.data()),
-                       sizeof(float) * scale_data.size()),
-            input_data_type);
+        scale_name = CreateInitializer<float>(constant_dims, scale_data,
```

Comment:
For float, you may need to pass `base::allow_nonunique_obj`, see https://source.chromium.org/chromium/chromium/src/+/main:base/containers/span.h;l=164;drc=f39c57f31413abcb41d3068cfb2c7a1718003cc5;bpv=1;bpt=1?q=base::as_byte_span&ss=chromium%2Fchromium%2Fsrc
```suggestion
        scale_name = CreateInitializer(constant_dims, base::as_byte_span(base::allow_nonunique_obj, scale_data),
```